### ISSUE TITLE
Entrypoint error on latest image on docker hub

### ISSUE THREAD

**Issue by cerealcable** [wastrachan/docker-bind#3]
```
bind2_1  | /docker-entrypoint.sh: line 38: /usr/sbin/named-checkconf: not found
```

I was using this image and suddenly there was a new image uploaded to docker hub and it's causing this error, however I am not seeing any code changes here, was there a deploy without a git push perhaps?

**Comment by cerealcable** [comment]
Also, maybe there should be version tags instead of _only_ latest, I had to manually tag an older image, thankfully I hadn't done a prune recently.

**Comment by wastrachan** [comment]
I'm able to reproduce this by pulling the latest image:
```
➜   docker pull wastrachan/bind:latest                      
latest: Pulling from wastrachan/bind

➜   docker run --rm -it wastrachan/bind

----------------------------------------
 Starting BIND9, using the following:   
                                        
     UID: 100                         
     GID: 101                         
----------------------------------------

/docker-entrypoint.sh: line 38: /usr/sbin/named-checkconf: not found
```

If I build from scratch, this issue is not present:
```
➜ make clean build
rm -f .bind.img
docker build -t wastrachan/bind:latest .
[...]

➜   docker run --rm -it wastrachan/bind

----------------------------------------
 Starting BIND9, using the following:   
                                        
     UID: 100                         
     GID: 101                         
----------------------------------------

25-Nov-2022 14:42:58.016 starting BIND 9.16.33 (Extended Support Version) <id:35e9c6e
```

I've noticed something very peculiar... the BIND version available in my local build is not the same as the BIND version from the CI build:

```
➜ docker run --rm -it --entrypoint "" wastrachan/bind:latest apk list | grep bind-9
bind-9.16.33-r0 x86_64 {bind} (MPL-2.0) [installed]

➜ docker run --rm -it --entrypoint ""  wastrachan/bind:dockerhub apk list | grep bind-9
bind-9.18.9-r0 x86_64 {bind} (MPL-2.0) [installed]
```

I do not know why two independent builds of the same docker file would yield two very different versions of `bind`... but it looks like `named-checkconf` has moved to a new location:

```
➜ docker run --rm -it --entrypoint ""  wastrachan/bind:dockerhub which named-checkconf 
/usr/bin/named-checkconf

➜ docker run --rm -it --entrypoint ""  wastrachan/bind:latest which named-checkconf    
/usr/sbin/named-checkconf
```

The entrypoint will be updated to account for this.

**Comment by wastrachan** [comment]
Fixed in [9b1a0be](https://github.com/wastrachan/docker-bind/commit/9b1a0beff07694b84e5c676be73b5cf419e4b3da)
Child issue: #4 (Investigate tagging image with latest alpine package version)
